js/oceanex.js

```diff
@@ -951,8 +1137,15 @@ module.exports = class oceanex extends Exchange {
             // exchange.secret = fs.readFileSync ('oceanex.pem', 'utf8')
             const jwt_token = this.jwt (request, this.encode (this.secret), 'RS256');
             url += '?user_jwt=' + jwt_token;
+        } else if (type === 'contract' && access === 'public') {
+            if (Object.keys (query).length) {
+                url += '?' + this.urlencode (query);
+            }
+        } else if (type === 'contract' && access === 'private') {
+            this.checkRequiredCredentials ();
+            headers['X-API-KEY'] = this.apiKey;
+            headers['Authorization'] = this.bearer;
```

Comment:
`this.bearer` is not documented anywhere on our side, let's use `this.token` instead:

```suggestion
            headers['Authorization'] = this.token;
```